site-scrapers/index.js

```diff
@@ -1,24 +1,21 @@
-const FamilyPracticeGroup = require("./FamilyPracticeGroup.js");
-const MAImmunizations = require("./MAImmunizations.js");
-const UMassAmherst = require("./UMassAmherst.js");
-const Hannaford = require("./Hannaford.js");
-const Harrington = require("./Harrington.js");
-const Curative = require("./Curative.js");
-const Atrius = require("./Atrius.js");
-const LynnTech = require("./LynnTech.js");
-const PriceChopper = require("./PriceChopper.js");
 
-let scrapers = [
-    FamilyPracticeGroup,
-    MAImmunizations,
-    UMassAmherst,
-    Hannaford,
-    Harrington,
-    Curative,
-    Atrius,
-    LynnTech,
-    PriceChopper
-];
+let scrapers = [];
+
+if(process.argv.length > 2){
+    let scraper;
+    for(let i = 2; i < process.argv.length; i++){
+        scraper = require(`./${process.argv[i]}.js`);
+        scrapers.push(scraper);
+    }
+}
+
+if (process.env.SCRAPERS) {
+    const scrapersList = process.env.SCRAPERS.split(' ');
+    scrapersList.map((id) => {
+        let scraper = require(`./${id}.js`);
+        scrapers.push(scraper);
+    });
+}
```

Comment:
I'm not a fan of this. I'd rather not have to keep a .env file (which is in .gitignore, and rightly so) updated in AWS Lambda just to make sure that all scrapers are running in production. There should be a hard-coded list that is invoked by default.

Reply:
See below.